README.md

```diff
@@ -89,9 +94,7 @@ List your noticed symptoms and get immediate help on the medicines that could be
 ### DFD explaining the data flow in the software:<br>
 ![DFd expalining data flow](https://files.gitter.im/muskankhedia/inED/moqup-_1_.png)
 
-## Technology used<br/>
+## :wrench: Tech Stack
 
-**Front-end**<br/>
-For front-end [AngularJS] is used.<br/>
-**Back-end**<br/>
-For back-end [GoLang] is used.
+* **Front-end:**  AngularJS
```

Comment:
**Front-end:** AngularJS for web and Electron + AngularJS for desktop app.

Reply:
@Harkishen-Singh Done.